pylint/testutils/output_line.py

```diff
@@ -139,13 +138,7 @@ def from_csv(
                 )
             raise IndexError
         except Exception:  # pylint: disable=broad-except
-            warnings.warn(
-                "Expected 'msg-symbolic-name:42:27:MyClass.my_function:The message:"
-                f"CONFIDENCE' but we got '{':'.join(row)}'. Try updating the expected"
-                f" output with:\npython tests/test_functional.py {UPDATE_OPTION}",
-                UserWarning,
-                stacklevel=2,
-            )
+            # We need this to not fail for the update script to work.
```

Comment:
Why don't we raise the warning anymore?

Reply:
We don't need the user warning because either we're going to fail with a message telling to update the output or we're already going to update the output. See the 8 added test cases. The UserWarning actually prevented the update in case of broken output.